src/kili/exceptions.py

```diff
@@ -1,11 +1,9 @@
 """Exceptions of the package."""
 from typing import Optional
 
-import graphql
-from gql.transport import exceptions
 
-TransportQueryError = exceptions.TransportQueryError
-GraphQLError = graphql.GraphQLError
+class GraphQLError(Exception):
+    """Raised when the GraphQL call returns an error"""
 
```

Comment:
our class was :
```
class GraphQLError(Exception):
    """
    Used when the GraphQL call returns an error
    """

    def __init__(self, error, batch_number=None):
        if isinstance(error, list):
            error = error[0]
        if isinstance(error, dict) and "message" in error:
            error_msg = error["message"]
        else:
            error_msg = str(error)

        if batch_number is None:
            super().__init__(f'error: "{error_msg}"')
        else:
            super().__init__(f'error at index {100*batch_number}: {error_msg}"')
```

Is it on purpose to drop the __init__ code ?

Reply:
the content of the graphqlerror will be printed in the graphql.GraphQLError or the gql.transports.exceptions.Transportqueryerror since we `raise GraphQLError() from err`

Reply:
is the message similar? in case clients were doing test with the graphQL error message

Reply:
no it's quite different, I'll revert to the previous init code. We'll have the error twice in the traceback but that's not an issue